src/Common/Misc/ConvertProxyProperty/index.js

```diff
@@ -92,14 +92,15 @@ export function proxyPropToProp(property, ui) {
   const depStatus = depList ? Boolean(Number(depList.pop())) : true;
   const depValue = depList ? depList.pop() : null;
   const depId = depList ? depList.join(':') : null;
-  const searchString = [ui.name].concat(property.value).join(' ').toLowerCase();
+  const searchString = [ui.name].concat(property.value, depList || []).join(' ').toLowerCase();
```

Comment:
I think it might be useful in the case of a proxy property, maybe in other cases as well.  I'll leave it up to you.